Exec/inputs_compare_subdomain

```diff
@@ -34,6 +34,7 @@ exchange_coupling = 1
 anisotropy_coupling = 0
 anisotropy_axis = 0.0 1.0 0.0
 DMI_coupling = 0
+FFT_solver = 0
```

Comment:
can you move this line so it's immediately after demag_coupling=1?
Also, please add FFT_solver=0 to any of the inputs files that have demag_coupling=1 in them.